src/core/api/disputes/disputes.types.ts

```diff
@@ -0,0 +1,30 @@
+import { Media } from '../media/media.types';
+import { Identity } from '../site/site.types';
+
+export interface IssueDisputePayload {
```

Comment:
Based on our naming convention, the name of this interface is better to be `DisputeReq` and the response interface is better to be named `DisputeRes`

Reply:
Dispute is such a general name
We have a type for:
- issuing
- responding
Do you have other suggestions?